.github/workflows/main.yml

```diff
@@ -16,6 +16,43 @@ on:
 
 jobs:
 
+  windows-arm64:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        CONFIGURATION: [Debug, RelWithDebInfo, Release]
+        CMAKE_GEN: ["Visual Studio 17 2022"] # Only version available on Actions w/ windows latest
+        FEATURE_DOC: [OFF, ON]
+        PACKAGE: [NO, YES]
```

Comment:
Do you want to not bother with the Configuration part of this matrix and just only build Release on CI?

Reply:
There are 6 arm64-related jobs at present. Not sure exactly what they all are. There only need to be 2: Debug and Release. The Release config should also build the package. There is no need for a separate job for this. I want Debug because I'm not doing VS 2022 builds on Appveyor at present.

I do not know how deploy works on Actions. On the other services you add a `deploy:` phase to the .yml file in which you can specify conditions on which to deploy as well as what and where to deploy. We need to do it only on tags and we want to deploy to GitHub Releases.

Reply:
I think I have a better approach that will clean this up